src/main/java/org/folio/roles/domain/entity/BasePolicyEntity.java

```diff
@@ -61,4 +62,12 @@ public class BasePolicyEntity extends Auditable {
   @JdbcTypeCode(SqlTypes.NAMED_ENUM)
   @Column(name = "logic", columnDefinition = "logic_type", insertable = false, updatable = false)
   private LogicType logic;
+
+  /**
+   * The source of the policy.
+   */
+  @Enumerated(EnumType.STRING)
+  @JdbcTypeCode(SqlTypes.NAMED_ENUM)
+  @Column(name = "source", columnDefinition = "source_type", updatable = false)
+  private SourceType source;
```

Comment:
there is specialization of `RoleEntity` called `LoadableRoleEntity`. it already has a field to classify roles: `type`
it'd consider moving `type` field to `RoleEntity`, because having two fields for similar purpose looks too excessive